tests/pipelines/ip_adapters/test_ip_adapter_stable_diffusion.py

```diff
@@ -387,3 +400,31 @@ def test_inpainting_sdxl(self):
         expected_slice = np.array([0.1398, 0.1476, 0.1407, 0.1442, 0.1470, 0.1480, 0.1449, 0.1481, 0.1494])
 
         assert np.allclose(image_slice, expected_slice, atol=1e-4, rtol=1e-4)
+
+    def test_controlnet_image_to_image(self):
```

Comment:
Do we need to add the slow test? @yiyixuxu

Reply:
we don't need to - do we prefer not too?